skfp/preprocessing/filters/pfizer.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP
+from rdkit.Chem.rdMolDescriptors import CalcTPSA
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class PfizerFilter(BaseFilter):
+    """
+    Pfizer 3/75 rule.
+
+    Based on observation that compounds exhibiting low clogP and high TPSA are roughly 2.5 times more likely to be free
+    of toxicity issues in the tested conditions [1]_ [2]_.
```

Comment:
Write full names of descriptors in this text, e.g. `low partition coefficient (clogP)`